ocaml/idl/datamodel_pool.ml

```diff
@@ -411,15 +411,40 @@ open Datamodel_types
       ~params:[String, "name", "A name to give the certificate";
                String, "cert", "The certificate"]
       ~allowed_roles:_R_POOL_OP
+      ~internal_deprecated_since:rel_next
       ()
 
+
+  let install_ca_certificate = call
+      ~in_oss_since:None
+      ~in_product_since:rel_next
+      ~name:"install_ca_certificate"
+      ~doc:"Install a TLS CA certificate, pool-wide."
+      ~params:[String, "name", "A name to give the certificate";
+               String, "cert", "The certificate"]
+      ~allowed_roles:_R_POOL_OP
+      ~internal_deprecated_since:rel_next
```

Comment:
I don't think this is already deprecated :)

Reply:
Oops - must have attached it to the wrong call.